assets/airports/kjfk.json

```diff
@@ -7,227 +8,623 @@
+            "airspace_class": "B",
+            "poly": [
+                [40.03355278, -73.71678889],
+                [40.06298611, -73.67481944],
+                [40.09182222, -73.63765278],
+                [40.21133583, -73.46443750],
+                [40.23725111, -73.43697444],
+                [40.40881250, -73.25295639],
+                [40.53523417, -73.08381694],
+                [40.55040944, -73.06578500],
+                [40.62937861, -72.96573806],
+                [40.66119667, -72.87438139],
+                [40.66933944, -72.85094444],
+                [40.70008417, -72.76214944],
+                [40.85703833, -72.76219444],
+                [40.90727111, -72.81511500],
+                [40.92884972, -72.93124639],
+                [40.90660806, -72.96964194],
+                [40.90394167, -72.97413722],
+                [40.90205944, -73.02891694],
+                [40.92364778, -73.13569528],
+                [40.86630556, -73.25213333],
+                [40.88573833, -73.28680056],
+                [40.91201944, -73.33351528],
+                [40.91872542, -73.46516244],
+                [40.87442861, -73.58006139],
+                [40.83117750, -73.69138556],
+                [40.83467944, -73.76042028],
+                [40.99880583, -74.10681333],
+                [41.00331278, -74.15769000],
+                [40.73953111, -74.26430806],
+                [40.66221500, -74.10978583],
+                [40.43231611, -74.08154222],
+                [40.42488611, -74.08773472],
+                [40.38402167, -74.18735028],
+                [40.30673139, -74.26767611],
+                [40.24996028, -74.33294528],
+                [40.15420583, -73.99997306],
+                [40.13753194, -73.89581917],
+                [40.06392917, -73.79860389]
+            ]
+        }
+    ],
     "fixes": {
-        "JFK"  : ["N40.6328889", "W73.7713889"],
-        "_RWY13RDEP": ["N40.56", "W73.660167"],
-        "_RWY13LDEP": ["N40.5945", "W73.607667"],
-        "NORTH": ["N41.3532606", "W73.7713889"],
-        "EAST" : ["N40.6290271", "W72.8258017"],
-        "SOUTH": ["N39.9124270", "W73.7713889"],
-        "WEST" : ["N40.6290271", "W74.7169761"],
-        "ROBER": ["N40.68546389", "W73.03261111"],
+        "_CRI176002": ["N40d34.82m0", "W73d52.99m0"],
+        "_CRI223003": [40.333333, -73.929167],
+        "_RWY13LSKORR062": ["N40d40.01m0", "W73d48.63m0"],
+        "_RWY13LSKORR063": ["N40d39.97m0", "W73d48.52m0"],
+        "_RWY13RSKORR057": ["N40d39.43m0", "W73d50.24m0"],
+        "_RWY13RSKORR058": ["N40d39.38m0", "W73d50.15m0"],
+        "_RWY22L01DME": [40.65954878329, -73.74368000403],
+        "_RWY22R01DME": [40.66484680209, -73.75212018818],
+        "_RWY31L01DME": [40.61947646339, -73.75297764097],
+        "_RWY31R01DME": [40.63520646392, -73.74046224393],
+        "ACK"  : ["N41d16.91m0", "W070d01.60m0"],
+        "AIR"  : ["N40d01.02m0", "W080d49.03m0"],
+        "ALB": ["N42d44.84m0", "W073d48.19m0"],
```

Comment:
Make sure to align the colons on those new fixes 😉

Reply:
Gah! Forgot 🤦‍♂ 

Fixed now.